2024.html

```diff
@@ -58,6 +58,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
               Thank you!
             -->
             <li>alifeee's <a href="https://alifeee.co.uk/">website</a>, <a href="https://blog.alifeee.co.uk/">blog</a>, and <a href="https://weeknotes.alifeee.co.uk/">weeknotes</a></li>
+            <li><a href="https://arne.me">Arne Bahlo</a></li>
```

Comment:
Minor: Please add leading slash in consistency with other entries. Thanks!

Reply:
Done!